packages/api/src/WithApiConnectors.tsx

```diff
@@ -0,0 +1,57 @@
+import { Connector } from '@syndesis/models';
+import * as React from 'react';
+import { IRestState } from './Rest';
+import { ServerEventsContext } from './ServerEventsContext';
+import { SyndesisRest } from './SyndesisRest';
+import { WithChangeListener } from './WithChangeListener';
+import { IChangeEvent } from './WithServerEvents';
+
+export interface IApiConnectorsResponse {
+  items: Connector[];
+  totalCount: number;
+}
+
+export interface IWithApiConnectorsProps {
+  disableUpdates?: boolean;
+  children(props: IRestState<IApiConnectorsResponse>): any;
+}
+
+export class WithApiConnectors extends React.Component<
+  IWithApiConnectorsProps
+> {
+  public changeFilter(change: IChangeEvent) {
+    return change.kind.startsWith('apiConnector');
```

Comment:
@gashcrumb do we have a list of values that `kind` can take somewhere?

Reply:
Actually I just *guessed* at the value of `change.kind` just to put something in there. Testing things confirms that my guess was not a good one. I looked at the angular codebase and couldn't find the answer (maybe you can @gashcrumb). On the upside, testing the change event in `WithExtensions` works fine.

Reply:
@riccardo-forina  I'll have to make one most likely.  Effectively it's not really bound to a set list.

@elvisisking I think it'd be `Connectors` as once an API client connector is created, it's just a connector.  But it has some special metadata, *however* I don't think this metadata is available within the IChangeEvent object you get back, typically they just have the kind and an ID of the changed thing.

Reply:
Looks like "connector" works @gashcrumb. When an API connector is created or deleted the ChangeEvent.kind is that. I'll make that change.